File: SDKs/Java/src/main/java/com/aspose/words/model/MetafileRenderingOptionsData.java
package com.aspose.words.model;

public class MetafileRenderingOptionsData {
  private String EmfPlusDualRenderingMode = null;
  private String RenderingMode = null;
  private Boolean UseEmfEmbeddedToWmf = null;
  /**
	 * getEmfPlusDualRenderingMode
	 * Gets String
	 * @return EmfPlusDualRenderingMode
	 */
  public String getEmfPlusDualRenderingMode() {
    return EmfPlusDualRenderingMode;
  }

	/**
	 * setEmfPlusDualRenderingMode
	 * Sets String
	 * @param EmfPlusDualRenderingMode String
	 */
  public void setEmfPlusDualRenderingMode(String EmfPlusDualRenderingMode) {
    this.EmfPlusDualRenderingMode = EmfPlusDualRenderingMode;
  }

  /**
	 * getRenderingMode
	 * Gets String
	 * @return RenderingMode
	 */
  public String getRenderingMode() {
    return RenderingMode;
  }

	/**
	 * setRenderingMode
	 * Sets String
	 * @param RenderingMode String
	 */
  public void setRenderingMode(String RenderingMode) {
    this.RenderingMode = RenderingMode;
  }

  /**
	 * getUseEmfEmbeddedToWmf
	 * Gets Boolean
	 * @return UseEmfEmbeddedToWmf
	 */
  public Boolean getUseEmfEmbeddedToWmf() {
    return UseEmfEmbeddedToWmf;
  }

	/**
	 * setUseEmfEmbeddedToWmf
	 * Sets Boolean
	 * @param UseEmfEmbeddedToWmf Boolean
	 */
  public void setUseEmfEmbeddedToWmf(Boolean UseEmfEmbeddedToWmf) {
    this.UseEmfEmbeddedToWmf = UseEmfEmbeddedToWmf;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class MetafileRenderingOptionsData {\n");
    sb.append("  EmfPlusDualRenderingMode: ").append(EmfPlusDualRenderingMode).append("\n");
    sb.append("  RenderingMode: ").append(RenderingMode).append("\n");
    sb.append("  UseEmfEmbeddedToWmf: ").append(UseEmfEmbeddedToWmf).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words_Cloud_SDK_for_.NET/src/Com/Aspose/Words/Model/MetafileRenderingOptionsData.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Words.Model {
  public class MetafileRenderingOptionsData {
    public string EmfPlusDualRenderingMode { get; set; }

    public string RenderingMode { get; set; }

    public bool UseEmfEmbeddedToWmf { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class MetafileRenderingOptionsData {\n");
      sb.Append("  EmfPlusDualRenderingMode: ").Append(EmfPlusDualRenderingMode).Append("\n");
      sb.Append("  RenderingMode: ").Append(RenderingMode).Append("\n");
      sb.Append("  UseEmfEmbeddedToWmf: ").Append(UseEmfEmbeddedToWmf).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
